tests/generator/test_errors.py

```diff
@@ -509,6 +509,19 @@ def test_device_bad_route_metric(self):
         - 192.168.14.2/24
         - 2001:FFfe::1/64''', expect_fail=True)
 
+    def test_device_bad_route_mtu(self):
+        self.generate('''network:
+  version: 2
+  ethernets:
+    engreen:
+      routes:
+        - to: 10.10.0.0/16
+          via: 10.1.1.1
+          mtu: -1
+      addresses:
+        - 192.168.14.2/24
+        - 2001:FFfe::1/64''', expect_fail=True)
```

Comment:
```suggestion
        - 2001:FFfe::1/64''', expect_fail=True)
        self.assertIn("invalid unsigned int value '-1'", err)
```
Could you please verify the explicit error message here?